src/rpc/blockchain.cpp

```diff
@@ -1679,7 +1710,13 @@ static UniValue getblockstats(const JSONRPCRequest& request)
             "  \"maxfeerate\": xxxxx,      (numeric) Maximum feerate (in satoshis per virtual byte)\n"
             "  \"maxtxsize\": xxxxx,       (numeric) Maximum transaction size\n"
             "  \"medianfee\": xxxxx,       (numeric) Truncated median fee in the block\n"
-            "  \"medianfeerate\": xxxxx,   (numeric) Truncated median feerate (in satoshis per virtual byte)\n"
+            "  \"feerate_percentiles\": [   (array of numeric) Feerates at the 10th, 25th, 50th, 75th, and 90th percentile weight unit (in satoshis per virtual byte)\n"
```

Comment:
nit: the other result keys are dictionary sorted in the help text. I suggest you follow the same convention here.